packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -0,0 +1,151 @@
+
+The header of requests sent by Okta will look like this, provided you configure an authorization header, as recommended, and do not define additional custom headers:
+
+```http
+Accept: application/json
+Content-Type: application/json
+Authorization: ${key}
+```
+
+The value sent in the Authorization header is a secret string you provide to Okta when you register your event hook. This string serves as an API access key for your service, and Okta provides it in every request, allowing your code to check for its presence as a security measure. (This is not an Okta authorization token, it is simply a text string you decide on.)
+
+### Security
+
+To secure the communication channel between Okta and your external service, HTTPS is used for requests, and support is provided for header-based authentication. Okta recommends that you implement an authentication scheme using the authentication header, to be used to authenticate every request received by your external service.
+
+### Your Service's Responses to Event Delivery Requests
+
+Your external service's responses to Okta's ongoing event delivery POST requests should all be empty, and should have an HTTP status code of 200 (OK) or 204 (No Content). As a best practice, you should return the HTTP response immediately, rather than waiting for any of your own internal process flows triggered by the event to complete.
+
+### Rate Limits
+
+Event hooks are limited to sending 100 000 events per 24-hour period. 
+
+## Event Hook Setup
+
+### Registering an Event Hook
+
+After implementing your external service, you need to tell Okta it exists and configure it, by making a POST request to the `/api/v1/eventHooks` API. You use the JSON payload of that request to provide information on the event hook you are registering, including:
+
+- the URI of your external service's endpoint
+- the list of specific event types you want to use the event hook for
+- the secret value Okta should send in the authorization header of requests
+
+The response you receive will confirm creation of the event hook and provide you with the unique ID value of the created event hook.
+
+### Verifying an Event Hook
+
+After registering the event hook, you need to trigger a one-time verification process by making a POST request to the `/api/v1/eventHooks/${eventHookId}/lifecycle/verify` API. Okta will then make the GET request to your endpoint.
+
+If verification is successful, the JSON payload of the response to your call to the `/verify` API will contain a property called `verificationStatus` set to `VERIFIED`.
+
+After your event hook is verified, it can take up to a minute for delivery of events to your endpoint to begin.
+
+## Sample Event Delivery Payload
+
+The following is an example of a JSON payload of a request from Okta to your external service:
+
+```json
+{
+  "eventType": "com.okta.event_hook",
+  "eventTypeVersion": "1.0",
+  "cloudEventsVersion": "0.1",
+  "eventID": "b5a188b9-5ece-4636-b041-482ffda96311",
```

Comment:
nit : We had some minor updates after our last conversation around this.
Changed `eventID` to `eventId`.
Added new top-level field ` "contentType": "application/json"`

Reply:
Changed capitalization in `eventId` and added `contentType` field.